src/kixi/hecuba/weather/service.clj

```diff
@@ -1,7 +1,22 @@
 (ns kixi.hecuba.weather.service
+  (:require [clojure.java.io :as io]
+            [clojure.tools.cli :refer [parse-opts]])
   (:gen-class))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(def cli-options
+  [["-c" "--config CONFIG" "Path to config edn file."
+    :default "config/settings.edn"]])
+
+(defn load-config [filename]
+  (with-open [r (io/reader filename)]
+    (read (java.io.PushbackReader. r))))
```

Comment:
Yes it makes a big difference.... first with `slurp` second time with the `load-config` function.

```kixi.hecuba.weather.service> (-main "-c/Users/jasonbell/work/git/kixi.hecuba.weather.service/config/settings.edn")
nil
nil
nil
;; => nil
kixi.hecuba.weather.service> (-main "-c/Users/jasonbell/work/git/kixi.hecuba.weather.service/config/settings.edn")```


http://localhost:8080/4/
Weather Station
;; => nil